fdm-app/app/components/blocks/fields-map.tsx

```
@@ -1,13 +1,31 @@
 import { useState } from "react";
-import { useFetcher, useNavigation } from "@remix-run/react";
+import { useFetcher } from "@remix-run/react";
 import { Map, GeolocateControl, NavigationControl, Source, Layer } from 'react-map-gl'
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)